opensafely/__init__.py

```diff
@@ -52,14 +52,14 @@ def show_help(**kwargs):
     # following recent documentation but has an old opensafely installed,
     # there's some hint as to why their invocation is failing before being told
     # by argparse.
-    if sys.argv[1] != "upgrade":
+    if len(sys.argv) < 2 or sys.argv[1] != "upgrade":
```

Comment:
Minor nitpick - I think `if len(sys.argv) == 1` is easier to understand, perhaps? it can never be len 0, right?

Reply:
Oh nevermind, I get it

Reply:
Yep, fair point. I've amended.